search-provider/eks-discovery-feed-provider.c

```diff
@@ -7,6 +7,8 @@
 
 #include <eos-knowledge-content.h>
 #include <eos-shard/eos-shard-shard-file.h>
+
+#include <stdlib.h>
```

Comment:
If this is for `rand()`, I think you can just use `g_random_int_range()` instead